docs/wowa/design-system-dialog/mobile-brief.md

```
@@ -0,0 +1,1084 @@
+
+### 거칠기
+
+- 모달/버튼: `roughness` (0.8)
+- X 마크: `roughness` (0.8)
+- 빗금 패턴: 0.5 (미묘함)
+
+## 패키지 의존성 확인
+
+### 모노레포 구조
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)